compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -3407,7 +3410,9 @@ public void visit(BLangQueryAction queryAction) {
             } else if (kind == NodeKind.LET_CLAUSE) {
                 parentEnv = typeCheckLetClause((BLangLetClause) clause, parentEnv);
             } else if (kind == NodeKind.WHERE) {
-                whereEnv = typeCheckWhereClause((BLangWhereClause) clause, doClause, parentEnv);
+                whereEnv = typeCheckWhereClause(((BLangWhereClause) clause).expression, doClause, parentEnv);
```

Comment:
Not formated here as well

Reply:
Formatting issues have been rectified in https://github.com/ballerina-platform/ballerina-lang/pull/23252